apps/web/src/containers/StakeNOTE/sidebars/cooldown.tsx

```diff
@@ -0,0 +1,142 @@
+import { useCallback, useContext } from 'react';
+import { NOTEContext } from '..';
+import { Box, useTheme } from '@mui/material';
+import { TransactionSidebar } from '@notional-finance/trade';
+import { FormattedMessage } from 'react-intl';
+import {
+  Body,
+  LabelValue,
+  CountdownCards,
+  Button,
+  Caption,
+} from '@notional-finance/mui';
+import {
+  useAccountDefinition,
+  useTransactionStatus,
+} from '@notional-finance/notionable-hooks';
+import {
+  Network,
+  SETTINGS_SIDE_DRAWERS,
+  getDateString,
+  getProviderFromNetwork,
+} from '@notional-finance/util';
+import {
+  useSideDrawerManager,
+  useSideDrawerState,
+} from '@notional-finance/side-drawer';
+import { SNOTEWeightedPool } from '@notional-finance/core-entities';
+
+export const CoolDown = () => {
+  const theme = useTheme();
+  const context = useContext(NOTEContext);
+  const { sideDrawerOpen } = useSideDrawerState();
+  const { isReadOnlyAddress, onSubmit } = useTransactionStatus(Network.mainnet);
```

Comment:
Can you show this component when a transaction is in progress?

![image](https://github.com/notional-finance/notional-monorepo/assets/977434/b0b3bf64-451d-45d5-9b16-12f9d1fadc8a)


Same thing on the StartCoolDown component in the portfolio

Reply:
I am trying to stake note to be able to actually test this but when I try to stake some of my note I get a gas limit error. Trying to find a work around

Reply:
Is it not triggering the token approval setting?

Reply:
I haven't been able to get the stake note flow to work. So I haven't been able to get to this step with a actual wallet that is able to trigger the transaction